AIPscan/Reporter/templates/reports.html

```diff
@@ -63,143 +63,136 @@
-              {% for puid in original_puids %}
-                <option value="{{ puid }}">{{ puid }}</option>
-              {% endfor %}
-            </select>
+             <span class="text-muted">AIPs containing original files in the selected format.
+              <br>Filter by storage location.</span>
           </td>
-          <td><a href="#"><button type="button" id="aipsByOriginalPUID" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
+          <td><a href="#"><button type="button" id="aipsByOriginalFormat" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
       </tr>
 
       {% if preservation_file_formats %}
           <tr>
               <td>
                   <strong>AIPs by preservation format</strong>
                   <br>
-                  <span class="text-muted">AIPs containing preservation copies in the selected format, filter by Storage Location</span>
-              </td>
-              <td>
-                <select id="preservationFormatSelect" method="GET" action="/" class="form-control" style="width:90%;">
+                   <select id="preservationFormatSelect" method="GET" action="/" class="form-control" style="width:90%;">
                   {% for file_format in preservation_file_formats %}
                     <option value="{{ file_format }}">{{ file_format }}</option>
                   {% endfor %}
                 </select>
               </td>
+              <td>
+                    <span class="text-muted">AIPs containing preservation copies in the selected format.
+                    <br>Filter by storage location.</span>
+              </td>
               <td><a href="#"><button type="button" id="aipsByPreservationFormat" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
           </tr>
       {% endif %}
 
+      <tr>
+          <td>
+              <strong>AIPs by PUID</strong>
+              <br>
+              <select id="originalPUIDSelect" method="GET" action="/" class="form-control" style="width:90%;">
+              {% for puid in original_puids %}
+                <option value="{{ puid }}">{{ puid }}</option>
+              {% endfor %}
+            </select>
+          </td>
+          <td>
+            <span class="text-muted">AIPs containing original files matching the selected PRONOM Unique Identifier (file format versions)
+              <br>Filter by storage location.</span>
+          </td>
+          <td><a href="#"><button type="button" id="aipsByOriginalPUID" class="btn btn-info" style="margin-left:5px; margin-bottom:5px;">Tabular</button></a></td>
+      </tr>
+
       {% if preservation_puids %}
           <tr>
               <td>
-                  <strong>AIPs by preservation format version</strong>
+                  <strong>AIPs by preservation copy PUID</strong>
```

Comment:
I have a slightly preference for "preservation derivative" or "preservation file" over "preservation copy" but I'll leave the choice up to you.